### ISSUE TITLE
[1.0.0-alpha.1] First element in dropdown always highlighted

### ISSUE THREAD

**Issue by Lolincolc** [Dogfalo/materialize#5382]
## Expected Behavior

Normally, items in a drop-down menu are only highlighted when the mouse is over.

## Current Behavior

Currently, in a dropdown menu, the first item in the list is constantly highlighted, even when the mouse is hovering other items in the list.
![Image of Yaktocat](https://i.gyazo.com/b60d572a240b13e4275656959e318d1a.gif)

## Steps to Reproduce (for bugs)

This bug can be seen in the documentation by clicking "Drop me!" button : http://next.materializecss.com/dropdown.html

## Your Environment

* Version used: 1.0.0-alpha.1
* Browser Name and version: Chrome Beta
* Operating System and version (desktop or mobile): Windows 10 Fall creator update

**Comment by EMembrillas** [comment]
Yes, this is a problem. If we want to use the dropdown to create a submenu element this is a nuisance.

**Comment by Buchx** [comment]
I concur, its frustrating.

**Comment by darylteo** [comment]
This goes into my materialize-css-hacks.scss.

```scss
.dropdown-content li:focus:not(:hover) {
    background-color: transparent;
}
````

Enjoy.

**Comment by acburst** [comment]
Fixed in 2084239b0d6c247ce01a65997cf449f6b5a0cbc7
